fourkitchens/sous-drupal-project
# Drupal9.1 compatibility

## Purpose:
- Drupal9.1 compatibility

## Testing:
- Pull down this branch
- If you are using an existing clone of the project check your settings.php file for `$databases['default']['default']` and if necessary update the lando config from referencing drupal8 to drupal9.
- `lando destroy -y` # Destroy any existing lando instances
- `lando start` # Upon build we now do a composer install, this tests the out of the box experience
- Verify that composer installs without error
- Verify that you can install a new database without errors
- Verify that you can set Emulsify as the default theme without errors
